requirements.txt

```diff
@@ -233,7 +231,7 @@ urllib3[secure]==1.26.8
     #   requests
     #   scout-apm
     #   sentry-sdk
-wagtail==3.0
+wagtail==4.1
```

Comment:
What do you think about refreshing this to `4.1.3`? https://docs.wagtail.org/en/stable/releases/4.1.3.html

Reply:
:+1: added in 18c1fccc91cb3567b88f147ff7e29465194cacc9